### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by pushpeshkarki** [smart-test-ti/SoloX#102] [General]
I am not sure, if there is any qctive development happening on this project? This one was looking very promising however i dont see any changes since last few months? is this abandoned?

**Comment by rafa0128** [comment]
> I am not sure, if there is any qctive development happening on this project? This one was looking very promising however i dont see any changes since last few months? is this abandoned?

of course not!!!

**Reply by pushpeshkarki** [reply]
Thanks Rafa
